frontend/src/components/samples/details/SampleDetailsPool.js

```diff
@@ -0,0 +1,217 @@
+                    // that was pooled.
+                    // TODO: https://206.12.92.46/issues/1295 (create separate parent sample column)
+                    <Link to={`/samples/${pooledSample.parent_sample_id}`}>{pooledSample.alias}</Link>
+                )
+            }  
+        },
+        {
+            title: "Volume Ratio",
+            dataIndex: "volume_ratio",
+            sorter: true,
+        },
+        {
+            title: "Kind",
+            dataIndex: "sample_kind",
+            sorter: true,
+            options: sampleKinds.items.map(x => ({ label: x.name, value: x.name })), // for getFilterProps
+            render: (_, sample) =>
+                <Tag>{sample.sample_kind ? sample.sample_kind : "POOL"}</Tag>,
+        },
+        {
+            title: "Library Type",
+            dataIndex: "library_type",
+            sorter: true
+        },
+        {
+            title: "Library Size",
+            dataIndex: "library_size",
+            sorter: true
+        },
+        {
+            title: "Index",
+            dataIndex: "index",
+            sorter: true,
+            render: (_, pooledSample) => {
+                return pooledSample.index && 
+                <Link to={`/indices/${pooledSample.index_id}`}>{pooledSample.index}</Link>
+            }
+        },
+        {
+            title: "Individual",
+            dataIndex: "individual_name",
+            sorter: true,
+            render: (_, pooledSample) => 
+            {
+                return pooledSample.individual_id ?
+                    <Link to={`/individuals/${pooledSample.individual_id}`}>{pooledSample.individual_name}</Link>
+                    : 
+                    <div>{pooledSample.individual_name}</div>
+            }
+                
+        },
+        {
+            title: "Collection Site",
+            dataIndex: "collection_site",
+            sorter: true
+        },
+        {
+            title: "Experimental Groups",
+            dataIndex: "experimental_groups",
+            sorter: false,
+            render: (_, pooledSample) => {
+                return pooledSample.experimental_groups && pooledSample.experimental_groups.length > 0 &&
+                    <>
+                        { pooledSample.experimental_groups.map( groupName => <div key={groupName}>{groupName}</div>) }
+                    </>
+            }
+        }
+    ].map((column) => ({ ...column, key: column.title }))
+}
+
+
+const SampleDetailsPool = ({sample}) => {
+    // Only display the pooled samples table if the sample is actually a pool.
+    // Otherwise, just display a message saying that there are no pooled samples to display.
+    return (
+        sample.is_pool ?
+            <PooledSamples sample={sample}/>
+        :
+            <NoPooledSamples/>
+    )
+}
+
+export const NoPooledSamples = () => {
+    return (
+        <div>This sample does not contain any pooled samples.</div>
```

Comment:
The message is a bit strange. I would go with "This is not a pooled sample." instead.